Automattic/wp-calypso
# Tooling/Building CalypsO: Bump `sharp` dependency version to 0.28 to allow building of Calypso on M1 macs

The 0.25.3 version of `sharp` seems to require `libvips` that has no build for M1 macs. This prevents Calypso from completing the `yarn install` step and subsequently is unable to start because of that.

Seems that version 0.28 of `sharp` solves this - https://github.com/lovell/sharp/issues/2460

This only affects native M1 flows, where the terminal/yarn/homebrew is not being ran through Rosetta.

#### Changes proposed in this Pull Request

* Bump `sharp` dependency version to 0.28

#### Testing instructions

* Apply PR
* Build Calypso
* Make sure it builds successfully
